client/extensions/woocommerce/app/store-stats/utils.js

```diff
@@ -148,14 +148,15 @@ export function getStartPeriod( date, unit ) {
  * @param {(string|number)} value - string or number to be formatted
  * @param {string} format - string of 'text', 'number' or 'currency'
  * @param {string} [code] - optional currency code
+ * @param {number} [decimals] - optional number of decimal places
  * @return {string|number} - formatted number or string value
  */
-export function formatValue( value, format, code ) {
+export function formatValue( value, format, code, decimals ) {
 	switch ( format ) {
 		case 'currency':
-			return formatCurrency( value, code );
+			return formatCurrency( value, code, { precision: decimals || 2 } );
```

Comment:
Should we check for the `precision` parameters here - https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/client/lib/format-currency/index.js#L11 and https://github.com/Automattic/wp-calypso/blob/b855b7a8a7e9f6a24e4fc87e8fe251cfd10603fd/client/lib/format-currency/currencies.js#L7

Reply:
Good find. We are not using a specified precision with currencies and I think its better to use the defaults found, so I'm going to remove this logic